lib/main_common.pm

```diff
@@ -1235,6 +1235,7 @@ sub load_x11tests {
         loadtest "x11/gnucash";
         loadtest "x11/hexchat";
         loadtest "x11/vlc";
+        loadtest "x11/remote_desktop/vino_screensharing_available";
```

Comment:
The test should also apply for SLE. Could you please test there and make sure to schedule then in a common branch?

Reply:
The `load_x11tests()` function is called from `/products/opensuse/main.pm` as well as from `/products/sle/main.pm` already. I'll just make sure that it works in SLE as well and provide you a test run.